subprojects/docs/src/docs/release/notes.md

```diff
@@ -28,31 +43,29 @@ Sometimes a user wants to declare the value of an exposed task property on the c
 
 The following examples exposes a command line parameter `--url` for the custom task type `UrlVerify`. Let's assume you wanted to pass a URL to a task of this type named `verifyUrl`. The invocation looks as such: `gradle verifyUrl --url=https://gradle.org/`. You can find more information about this feature in the [user guide](userguide/custom_tasks.html#sec:declaring_and_using_command_line_options).
 
-```
-import org.gradle.api.tasks.options.Option;
-
-public class UrlVerify extends DefaultTask {
-    private String url;
-
-    @Option(option = "url", description = "Configures the URL to be verified.")
-    public void setUrl(String url) {
-        this.url = url;
+    import org.gradle.api.tasks.options.Option;
```

Comment:
Thank you for catching this bug 👍

Reply:
What's bug here? Can we not process "```" instead of indented code examples?

Reply:
That's correct. See https://docs.gradle.org/nightly/release-notes.html#default-jacoco-version-upgraded-to-0.8.0

Reply:
Good to know. Thanks.